### PR TITLE
RavenDB-20938 Apply warning from RavenDB-19967 to v6.0 Studio

### PR DESCRIPTION
**Pull request by kalczur**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-20938/Apply-warning-from-RavenDB-19967-to-v6.0-Studio

### Type of change

- New feature

### How risky is the change?

- Not relevant

### Backward compatibility

- Non breaking change

### Is it platform specific issue?

- No

### Documentation update

- No documentation update is needed 

### Testing by Contributor

- It has been verified by manual testing

### Testing by RavenDB QA team

- No special testing by RavenDB QA team is needed

### Is there any existing behavior change of other features due to this change?

- No

### UI work

image

### CODE REVIEW

File: src/Raven.Studio/typescript/components/pages/database/tasks/shared/OngoingTaskOperationConfirm.tsx [hunk 1]
```diff
@@ -105,6 +105,15 @@ export default function OngoingTaskOperationConfirm(props: OngoingTaskOperationC
                         {idx < taskGroups.length - 1 && <hr className="m-0" />}
                     </div>
                 ))}
+
+                {type === "disable" && (
```
Comment (ArieSLV) [inline src/Raven.Studio/typescript/components/pages/database/tasks/shared/OngoingTaskOperationConfirm.tsx]: Will this warning appear if we disable the `Subscription` ongoing task?
https://issues.hibernatingrhinos.com/issue/RavenDB-19967/Notification-warning-about-tombstone-cleaner-subscribers-disabled.#focus=Comments-67-590666.0-0

Comment (kalczur) [reply src/Raven.Studio/typescript/components/pages/database/tasks/shared/OngoingTaskOperationConfirm.tsx]: Now we allow changing the state simultaneously for multiple tasks (regardless of type). I can hide the warning if only subscription tasks are on the list to disable.

Comment (ArieSLV) [reply src/Raven.Studio/typescript/components/pages/database/tasks/shared/OngoingTaskOperationConfirm.tsx]: As I see the situation, if we want to be as precise as possible in our warnings, we need to handle four different states:

1. When only one `Subscription` task is selected (no warning needed):

```
// no warning message to be shown
```

2. When only one non-`Subscription` task is selected, and a warning is needed:

```
Please note, <strong>disabling</strong> this task will lead to continuous tombstone accumulation until it is re-enabled or deleted, resulting in increased disk space usage.
```

3. When multiple tasks are selected, and the warning applies to all:

```
Please note, <strong>disabling</strong> these tasks will cause continuous tombstone accumulation until they are re-enabled or deleted, leading to an increase in disk space usage.
```

4. When multiple tasks are selected, and one of them is a `Subscription`:

```
Please note, <strong>disabling</strong> the selected tasks (excluding <strong>Subscription</strong> tasks) will lead to continuous tombstone accumulation until they are re-enabled or deleted, which will increase disk space usage. This warning does not apply to Subscription tasks.
```

Is this feasible?

Comment (kalczur) [reply src/Raven.Studio/typescript/components/pages/database/tasks/shared/OngoingTaskOperationConfirm.tsx]: Sure, I can do it this way.

Comment (kalczur) [reply src/Raven.Studio/typescript/components/pages/database/tasks/shared/OngoingTaskOperationConfirm.tsx]: Done https://github.com/ravendb/ravendb/pull/16960/commits/c88d630c462e0be36f02e6fb7bbd5dfe343fec66

Comment (ArieSLV) [reply src/Raven.Studio/typescript/components/pages/database/tasks/shared/OngoingTaskOperationConfirm.tsx]: Yeah, looks good now!

Two `subscription` tasks and `backup` task to disable:
image

`Backup` task and `RavenDB ETL` task to disable:
image

Only `backup` task to disable:
image

Only `subscription` task to disable:
image

**Review by ArieSLV — APPROVED** [review]

**Review by ml054 — APPROVED** [review]

**Pull request merged**